exhibitor-core/src/main/java/com/netflix/exhibitor/core/azure/AzureClientImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.netflix.exhibitor.core.azure;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.blob.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+public class AzureClientImpl implements AzureClient {
+    private final Logger log = LoggerFactory.getLogger(getClass());
+    private final String storageConnectionString;
+
+    public AzureClientImpl(AzureCredential credentials) {
+        this.storageConnectionString =
+            "DefaultEndpointsProtocol=http" +
+            ";AccountName=" + credentials.getAccountName() +
+            ";AccountKey=" + credentials.getAccountKey();
+    }
+
+    @Override
+    public CloudBlobClient getClient() throws Exception {
```

Comment:
do you really that wide a blanket?
most code should *never* throw `Exception` - if you really have to spread such a wide net, I'd suggest you catch, do some logging, then re-throw a narrower exception (with a meaningful error message)

(this applies to all the methods in this class, actually)